deploy/olm-catalog/opendatahub/1.0.10/manifests/opendatahub-operator.v1.0.10.clusterserviceversion.yaml

```diff
@@ -0,0 +1,344 @@
+
+    Open Data Hub operator allows users to install and manage Open Data Hub and Kubeflow components. Users can mix and match tools from each project to fulfill the needs of their use case. Each of the
+    projects share some components, but can be mostly seen as an extension of each other to provide a complete solution for both novice and skilled enterprise users.
+
+    ### Components
+    * Spark Thrift Server - A service that allows JDBC clients run Spark SQL queries.
+    * Hue v4.8.0 - A service that provides data exploration on Hive and S3 buckets.
+    * Open Data Hub Dashboard v1.0  - A web dashboard that displays installed Open Data Hub components with easy access to component UIs and documentation
+    * JupyterHub v0.2.8 - Open source multi-user notebook platform w/ GPU support.
+    * Elyra v0.0.7 - JupyterLab notebooks with support for AI workflows
+    * Ceph Nano v0.7 - Minimal Object Storage provided by Ceph for development use
+    * Apache Spark v2.4.5 - Unified analytics engine for large-scale data processing
+    * Prometheus v2.16.0 - Monitoring and alerting tool
+    * Grafana v7.1.1 - Data visualization and monitoring
+    * Airflow v1.10.11 - Workflow management
+    * Seldon v1.2.0 - Open source platform for deploying machine learning models
+    * Argo v2.12.5 - Container-native Workflow Engine
+    * Apache Superset v0.30.0 - Open source application for data exploration and visualization
+    * Apache Kafka v2.7.0 - The open source stream processing platform
+    To install one or multiple of these components use the default KfDef provided with the operator.
+
+    ### Kubeflow Components
+    ODH 1.0 supports Kubeflow v1.2.0 and some components such as KF Serving and KFP on Tekton from master branch. This will be updated when Kubeflow v1.3.0 is released. To install Kubeflow v1.2.0 components with KFP using Argo please use this example [KfDef](https://raw.githubusercontent.com/kubeflow/manifests/v1.2-branch/kfdef/kfctl_openshift.v1.2.0.yaml). To install KF Serving with KFP on Tekton please use this example [KfDef](https://raw.githubusercontent.com/kubeflow/manifests/v1.3-branch/distributions/kfdef/kfctl_openshift.master.kfptekton.yaml)
+
+    ### Available Channels
+
+    #### Beta
+
+    Channel `beta` offers the latest version of Open Data Hub operator. It is based on Kubeflow Operator and Kustomize for deployment configuration. This version is compatible with [Open Data Hub manifests](https://github.com/opendatahub-io/odh-manifests/) as well as [Kubeflow manifests](https://github.com/opendatahub-io/manifests). This channel will offer new components and features.
+
+    To deploy Kubeflow, use [this example](https://github.com/kubeflow/manifests/blob/master/kfdef/kfctl_openshift.v1.2.0.yaml).
+
```

Comment:
We need to update the links to kfdef: https://github.com/kubeflow/manifests/blob/master/distributions/kfdef/kfctl_openshift.v1.2.0.yaml 
https://github.com/kubeflow/manifests/blob/master/distributions/kfdef/kfctl_openshift.master.kfptekton.yaml

Reply:
1. I used the raw links if anyone wanted to copy and paste it directly but I can change it to the "pretty" github link
2. I have both kfdef urls tagged since the `master` branch will move/update/delete them at anytime.